source/installing.rst

```diff
@@ -0,0 +1,233 @@
+
+The disadvantage of using a Minor Release repository is that you are locked in this particular release. When potentially critical fixes are released in a later minor version of the database, you will not be prompted for an upgrade by the package manager of your operating system. You would need to change the configured repository in order to install the upgrade.
+
+Prerequisites
+================================================================================
+
+Install |percona-release| if you haven't done it already. Otherwise, update it to the latest version. Please refer to `Percona repositories documentation <https://www.percona.com/doc/percona-repo-config/percona-release.html>`_ for installation or update instructions.
+
+Enable Percona repository       
+================================================================================
+
+We recommend to use the ``setup`` subcommand of |percona-release| to enable the desired repository.
+ 
+* For |PS|-based deployment, use the following command:
+
+  .. code-block:: bash
+
+     $ sudo percona-release setup pdps-8.0
+
+* For |PXC|-based deployment, use the following command:
+
+  .. code-block:: bash
+
+     $ sudo percona-release setup pdpxc-8.0
+
+.. hint::
+
+   To enable the minor version repository, use the following comand:
+
+   .. code-block:: bash
+   
+      $ #For PS-based deployment
+      $ sudo percona-release setup pdps-8.0.19
+      $ #For PXC-based deployment
+      $ sudo percona-relase setup pdpxc-8.0.19
+
+Make sure to run |percona-release| as root or via |sudo|. For the
+sake of convenience, all commands that require elevated privileges start with
+|sudo| in the following procedures.
+
+Install |pdmysql|
+================================================================================
+
+Install the selected deployment of |pdmysql| using the commands of the package manager of your operating system.
+
+.. important::
+
+   Make sure to run all commands as root or via |sudo|.
+
+Install |PS|-based deployment on Debian / Ubuntu
+--------------------------------------------------------------------------------
+
+1. Install |PS|:
+   
+   .. code-block:: bash
+  
+      $ sudo apt-get install percona-server
+
+#. Install the components. Use the commands below to install the required components: 
+   
+   .. code-block:: bash
+   
+      $ #Install Percona XtraBackup
+      $ sudo apt-get install percona-xtrabackup-80
+      $ #Install Percona Toolkit
+      $ sudo apt-get install percona-toolkit
+      $ #Install Orchestrator
+      $ sudo apt-get install percona-orchestrator
```

Comment:
For orchestrator we have a 3 packages:
        - percona-orchestrator-cli
        - percona-orchestrator-client
        - percona-orchestrator

Reply:
Do we need to explicitly mention all three?

Reply:
Depends on what do you need. I think all this packages needed, but maybe in documentation need to split it on 3 steps of installation process. Not sure how better.